frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -17,7 +17,7 @@ import {
   QueryWrapper,
 } from "../commonComponents/QueryWrapper";
 
-export const testResultQuery = gql`
+const testResultQuery = gql`
```

Comment:
Note: reusing gql queries means that we lose traceability in our logging. Currently in the azure dashboard it is impossible to know if a `GetFacilityResults` query is coming from the results page or from the Conduct Test Page

Reply:
The Conduct Test query just needs the one test, so it properly needs its own query returning just one test. It seems like our logging should be able to determine where the query came from, but if it's just backend then yeah it has no way of knowing.